src/Static.jl

```diff
@@ -517,13 +517,11 @@ Base.xor(x::Union{Integer, Missing}, ::StaticInteger{Y}) where {Y} = xor(x, Y)
 Base.:(!)(::True) = False()
 Base.:(!)(::False) = True()
 
-Base.all(::Tuple{Vararg{True}}) = true
-Base.all(::Tuple{Vararg{Union{True, False}}}) = false
-Base.all(::Tuple{Vararg{False}}) = false
+Base.all(::Tuple{True, Vararg{True}}) = true
+Base.all(::Tuple{Union{True,False}, Vararg{Union{True, False}}}) = false
```

Comment:
AFAICT this will fix the formatting issue:
```suggestion
Base.all(::Tuple{Union{True, False}, Vararg{Union{True, False}}}) = false
```